server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/remoteDelivery/RemoteDeliveryConfiguration.java

```diff
@@ -102,11 +103,25 @@ public RemoteDeliveryConfiguration(MailetConfig mailetConfig, DomainList domainL
         DelaysAndMaxRetry delaysAndMaxRetry = computeDelaysAndMaxRetry(mailetConfig);
         maxRetries = delaysAndMaxRetry.getMaxRetries();
         delayTimes = delaysAndMaxRetry.getExpandedDelays();
-        smtpTimeout = computeSmtpTimeout(mailetConfig);
-        connectionTimeout = computeConnectionTimeout(mailetConfig);
-        dnsProblemRetry = computeDnsProblemRetry(mailetConfig);
+        smtpTimeout = MailetUtil.integerConditionParser()
```

Comment:
it probably was better before because details were hidden behind a domain name "computeSmtpTime", etc.
Why did you decided to remove these methods ?